readme.md

```diff
@@ -985,7 +985,24 @@ Type: `string`
 
 The passphrase to decrypt the `options.https.key` (if different keys have different passphrases refer to `options.https.key` documentation).
 
-##### Examples for `https.key`, `https.certificate` and `https.passphrase`
+##### https.pfx
+
+Type: `string | Buffer | (string | Buffer | object)[]`
```

Comment:
```suggestion
Type: `string | Buffer | Array<string | Buffer | object>`
```

This is more readable.

Reply:
Why is `object` only supported in an array? Is that correct or a typo?

Reply:
And yes, it is correct that `object` is only supported in the array form. I updated the README to clarify